src/Build/Graph/ProjectGraph.cs

```diff
@@ -396,6 +426,14 @@ private void LoadGraph(ConcurrentQueue<ConfigurationMetadata> projectsToEvaluate
                     evaluationWaitHandle.WaitOne();
                 }
             }
+            if(exceptionsInTasks.Count != 0)
```

Comment:
```suggestion
            if (exceptionsInTasks.Count != 0)
```
I gotta try the new feature.